papermill/tests/test_parameterize.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import unittest
+
+from ..api import read_notebook
+from ..execute import _parameterize_notebook
+from . import get_notebook_path
+
+PYTHON = 'python2' if sys.version_info[0] == 2 else 'python3'
+
+
+class TestNotebookParametrizing(unittest.TestCase):
+    def test_preserving_tags(self):
+        # test that other tags on the parameter cell are preserved
+        test_nb = read_notebook(get_notebook_path("simple_execute.ipynb"))
+        test_nb.node.cells[0]['metadata']['tags'].append('some tag')
+
+        _parameterize_notebook(test_nb.node, PYTHON, {'msg': 'Hello'})
+
+        cell_zero = test_nb.node.cells[0]
+        self.assertTrue('some tag' in cell_zero.get('metadata').get('tags'))
+
+        cell_one = test_nb.node.cells[1]
+        self.assertTrue('some tag' in cell_one.get('metadata').get('tags'))
```

Comment:
Hmm do you think having all tags replicated in both cells may cause problems for certain workflows? I.e. workflows that search for a particular flag. I think maybe we should leave tags on the original cell and just move parameters to the new cell, leaving default_parameters in its place? Thoughts?

Reply:
I couldn't decide whether as a user I'd expect tags to "propagate" or not :man_shrugging: 

Sounds like you'd expect them not to propagate so going to change it :)